python/src/aiconfig/editor/client/src/components/prompt/prompt_outputs/PromptOutputsRenderer.tsx

```diff
@@ -103,5 +103,11 @@ const OutputRenderer = memo(function Output({ output }: { output: Output }) {
 });
 
 export default memo(function PromptOutputsRenderer({ outputs }: Props) {
-  return outputs.map((output, i) => <OutputRenderer key={i} output={output} />);
+  return (
+    <Flex direction="column" className="outputContainer">
```

Comment:
This has no actual difference in its own, but adding this container element is good since we can use the className for modifying styles as needed (also seems 'correct' to have the outputs all under a root element containing them)